backend/src/test/java/de/neuefische/backend/service/UtilServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.dto.AppointmentDto;
+import de.neuefische.backend.dto.DoctorDto;
+import de.neuefische.backend.model.Appointment;
+import de.neuefische.backend.model.Doctor;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class UtilServiceTest {
+
+    private final UtilService utilService = new UtilService();
+
+    @Test
+    void mapAppointmentDtoToAppointmentTest() {
+        // GIVEN
+        AppointmentDto appointmentDto = AppointmentDto.builder()
+                .date("2021-11-08")
+                .reasonForVisit("checkup")
+                .build();
+
```

Comment:
https://stackoverflow.com/questions/38941580/how-to-write-junit-test-cases-for-row-mapper-class-in-spring